### PR TITLE
Gettextize Offline_Backup.md, prove-payment.md and remote_node_gui.md

### PR DESCRIPTION
**Pull request by erciccione**
fourth batch of gettextized user guides (third batch: #1531)

Gettextized

- Offline_Backup.md
- prove-payment.md
- remote_node_gui.md

Part of #1270

### CODE REVIEW

File: _i18n/tr/resources/user-guides/weblate/prove-payment.po [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+
+#. type: Plain text
+#: _i18n/en/resources/user-guides/prove-payment.md:70
+msgid ""
+"The information supplied by Alice plugs neatly instead of the "
+"placeholders. monero-wallet-cli will use the transaction key to decode the "
+"transaction, and display how much this particular transaction sent to this "
+"address. Obviously, Charlie will want to double check with Bob the address "
+"is really his - same as with Bitcoin."
+msgstr ""
```
Comment (wirrickelliot) [inline _i18n/tr/resources/user-guides/weblate/prove-payment.po:205]: Ditto

File: _i18n/zh-cn/resources/user-guides/weblate/prove-payment.po [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+
+#. type: Plain text
+#: _i18n/en/resources/user-guides/prove-payment.md:70
+msgid ""
+"The information supplied by Alice plugs neatly instead of the "
+"placeholders. monero-wallet-cli will use the transaction key to decode the "
+"transaction, and display how much this particular transaction sent to this "
+"address. Obviously, Charlie will want to double check with Bob the address "
+"is really his - same as with Bitcoin."
+msgstr ""
```
Comment (wirrickelliot) [inline _i18n/zh-cn/resources/user-guides/weblate/prove-payment.po:205]: Ditto

**Review by wirrickelliot — CHANGES_REQUESTED** [review]
`_i18n/zh-tw/resources/user-guides/weblate/prove-payment.po` is missing.

**Comment by erciccione** [comment]
All fixed.

**Review by wirrickelliot — APPROVED** [review]

**Pull request merged**